packages/system/src/border/border.test.ts

```diff
@@ -22,23 +23,17 @@ import {
   borderBottomRightRadius,
 } from './border';
 
-const borders = [0, 1, 2];
-
-[borders.zero, borders.small, borders.medium] = borders;
-
-const radii = [0, 4, 8, 9999];
-
-[radii.sharp, radii.semiRounded, radii.rounded, radii.circle] = radii;
-
-const colors = {
-  vibin: '#D8385E',
-  hope: '#1D856C',
+const borders = {
+  zero: 0,
+  small: 1,
+  medium: 2,
 };
 
 const theme = {
   yoga: {
     borders,
     colors,
```

Comment:
Use a simpler mock to colors, like
```suggestion
    colors: {}
```

Reply:
I had to create the colors object with vibin and hope because these two colors are being used in the test